apps/core/scenes/search/Search.js

```diff
@@ -150,27 +27,8 @@ class Search extends React.Component<Props> {
           <View style={styles.illustrationWrapper}>
             <Illustration name="Boarding" style={styles.boardingIllustration} />
           </View>
-          <SearchFormModes />
-
-          <View style={desktopLayout && styles.inputsDesktop}>
-            <Placepickers />
-            <Datepickers />
-
-            <View
-              style={[styles.bottom, desktopLayout && styles.bottomDesktop]}
-            >
-              <Button
-                onPress={this.handleSubmitPress}
-                label="Search"
-                rightIcon={
-                  Platform.OS === 'web' ? <Icon name="chevron-right" /> : null
-                }
-              />
-            </View>
-          </View>
-          <View style={styles.bottomSpacer} />
```

Comment:
I think we should keep this spacer as part of the Search screen. It simply adds bottom padding on web to prevent sticking inputs to bottom edge on small screens and centers content on mobile.
It is related only to this screen and doesn't need to be present anywhere else. On results screen we have it disabled for now, but quite unintentionally by current set of rules.